src/main/java/com/easypost/model/Event.java

```diff
@@ -77,39 +31,39 @@ public void setId(final String id) {
     }
 
     /**
-     * Get the description of this Event.
+     * Get the mode of this Event.
      *
-     * @return Event's description
+     * @return Event's mode
      */
-    public String getDescription() {
-        return description;
+    public String getMode() {
+        return mode;
     }
 
     /**
-     * Set the description of this Event.
+     * Set the mode of this Event.
      *
-     * @param description Event's description
+     * @param mode Event's mode
      */
-    public void setDescription(final String description) {
-        this.description = description;
+    public void setMode(final String mode) {
+        this.mode = mode;
     }
 
     /**
-     * Get the mode of this Event.
+     * Get the description of this Event.
```

Comment:
The getters/setters don't match the same order of the properties.

What do you suggest here? We can make the getters/setters alphabetical (will mismatch the property order) or ensure the getter/setter order matches the property ordering for consistency.

Reply:
I was wondering that myself. I don't think there's a way to match the order of properties and the getter/setters (I was hoping it did, but seems it didn't).

The former, alphabetizing them, is enforceable by .editorconfig

Reply:
Alphabetical it is for now. We can revisit this when we clean up inherited properties in the future.